service/src/main/resources/config/verily/sdd/expanded/entity/person.json

```diff
@@ -123,8 +123,8 @@
     "dataType" : "INT64",
     "displayHint" : {
       "type" : "RANGE",
-      "minVal" : 1799.0,
-      "maxVal" : 2993.0
+      "minVal" : 1.0,
+      "maxVal" : 5227737.0
```

Comment:
This is interesting -- I would have expected the min to be `0.0` and the max to be unchanged. Are these the right values in the tables?

Reply:
I re-expanded and these are correct now (0 - 2993). Not sure what happened earlier.